src/components/withPrepareCentralPaneScreen/index.tsx

```diff
@@ -0,0 +1,9 @@
+import type {ComponentType} from 'react';
+
+/**
+ * This HOC is dependent on platform. On native platforms screens that aren't already dipslayed in the navigation stack should be freezed to prevent from unnecessary rendering.
+ * It's handled this way only on mobile platforms, because on web more than one screen is displayed in a wide layout, so these screens shouldn't be freezed.
```

Comment:
```suggestion
 * It's handled this way only on mobile platforms because on the web, more than one screen is displayed in a wide layout, so these screens shouldn't be frozen.
```